lib/que/metrics.rb

```diff
@@ -7,62 +7,32 @@
 
 module Que
   class Metrics
-    def initialize(labels: {})
```

Comment:
Separate out the registration of the prometheus metrics from the creation of the Que worker. In future we may decide to make this a module initialisation step, but for now we rely on the idempotence of the registration and have each worker register the metrics before use.